mbq/ranch/signal_handlers/rollbar_handlers.py

```diff
@@ -0,0 +1,22 @@
+import rollbar
+from celery.signals import task_failure, task_rejected, task_unknown
+
+from ..lib.error_handling import log_errors_and_send_to_rollbar
+
+
+@task_failure.connect
+@log_errors_and_send_to_rollbar
```

Comment:
Why do we need the decorator and the rollbar in the body of the function?

Reply:
well, we probably don't _need_ the decorator in this case, but they serve different purposes. The decorator catches errors that occur in the signal handlers themselves. But the signal handlers here serve to rollbar errors that occur in the running tasks.

Reply:
Yeah I just meant that if `rollbar.report_exc_info` fails then we're not likely to have any luck calling `rollbar.report_exc_info` again. The logging is +1 tho